docs/components/modeler/desktop-modeler/telemetry/telemetry.md

```diff
@@ -4,6 +4,12 @@ title: Telemetry
 description: "You can opt-in the collection of telemetry data when using the desktop modeler. This data will be used to better understand how the application is used and to improve it based on data."
 ---
 
+:::caution
```

Comment:
Why should we add this? I think this is more confusing than helping.

Reply:
Do you think newbies to Camunda will understand the Camunda Platform 8 value is "Camunda Cloud"? I'm hoping this eliminates confusion for folks who didn't previously know about Camunda Cloud.

Reply:
Please keep in mind that this is pure technical reference documentation. There is nothing a user needs to do. We only clearly state what kind of data we send for usage tracking.

Reply:
Alternatively we could just make the example data shown more generic, so that the question does not even come up. E.g.,

```json
{
  "engineProfile": {
    "executionPlatform": "...", // <- platform used in current diagram
    "executionPlatformVersion": "..." // <- platfrom version used in current diagram
  }
}
```

Reply:
Actually I think that would be best. I will add this on top of this PR as a commit if you allow

Reply:
Proposed via https://github.com/camunda-cloud/camunda-cloud-documentation/pull/721/commits/98f6cd82f5b3dc5a03b6cf2130c0165af77b4d96